privilege/privilege.go

```diff
@@ -42,7 +42,7 @@ type Manager interface {
 	// If table is not "", check global/db/table scope privileges.
 	// priv should be a defined constant like CreatePriv, if pass AllPrivMask to priv,
 	// this means any privilege would be OK.
-	RequestVerification(activeRole []*auth.RoleIdentity, db, table, column string, priv mysql.PrivilegeType) bool
+	RequestVerification(activeRole []*auth.RoleIdentity, db, table, column string, priv mysql.PrivilegeType, err error) bool
```

Comment:
Would it not be better to replace the return type with error, instead of using an in-parameter as for return values? (If you decide to change the API)